magellan-library/src/main/java/com/wealthfront/magellan/core/Journey.kt

```diff
@@ -12,6 +12,4 @@ public abstract class Journey<V : ViewBinding>(
 ) : Step<V>(createBinding) {
 
   protected var navigator: LinearNavigator by lifecycle(LinearNavigator { viewBinding!!.container() })
-
-  override fun toString(): String = this.javaClass.simpleName
```

Comment:
This was just so that we don't have a memory address printed in the logs during navigation but i've actually found them pretty useful so removing it so that we can have the class name + memory address